api/start.sh

```diff
@@ -13,12 +13,17 @@ handle_signal() {
 # Trap termination signals and call the handle_signal function
 trap 'handle_signal' SIGINT SIGTERM
 
-# Start taskiq as a background process
+# Start taskiq worker as a background process
```

Comment:
I don't think you need to change that file, especially since it's not used in production.
Also I don't think you need a scheduler process since we only use dynamic scheduling -> https://taskiq-python.github.io/guide/scheduling-tasks.html#dynamic-scheduling

Reply:
I'll double check but when I tested the scheduled task without a scheduler, the delayed task were not running.

Reply:
As discussed, the start script is only used on preview or quick start local dev. No need to have a scheduler in this env. I'll remove !